File: pyg_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/SellerService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbSeller;
import entity.PageResult;

import java.util.List;

/**
 * 服务层接口
 *
 * @author Administrator
 */
public interface SellerService {

    /**
     * 返回全部列表
     *
     * @return
     */
    List<TbSeller> findAll();


    /**
     * 返回分页列表
     *
     * @return
     */
    PageResult findPage(int pageNum, int pageSize);


    /**
     * 增加
     */
    void add(TbSeller seller);


    /**
     * 修改
     */
    void update(TbSeller seller);


    /**
     * 根据ID获取实体
     *
     * @param id
     * @return
     */
    TbSeller findOne(Long id);


    /**
     * 批量删除
     *
     * @param ids
     */
    void delete(Long[] ids);

    /**
     * 分页
     *
     * @param pageNum  当前页 码
     * @param pageSize 每页记录数
     * @return
     */
    PageResult findPage(TbSeller seller, int pageNum, int pageSize);

}

File: pyg_manager_web/src/main/webapp/js/service/sellerService.js
//服务层
app.service('sellerService', function ($http) {

    //读取列表数据绑定到表单中
    this.findAll = function () {
        return $http.get('../seller/findAll');
    };
    //分页
    this.findPage = function (page, rows) {
        return $http.get('../seller/findPage/' + page + '/' + rows);
    };
    //查询实体
    this.findOne = function (id) {
        return $http.get('../seller/findOne/' + id);
    };
    //增加
    this.add = function (entity) {
        return $http.post('../seller/add', entity);
    };
    //修改
    this.update = function (entity) {
        return $http.post('../seller/update', entity);
    };
    //删除
    this.dele = function (ids) {
        return $http.get('../seller/delete/' + ids);
    };
    //搜索
    this.search = function (page, rows, searchEntity) {
        return $http.post('../seller/search/' + page + "/" + rows, searchEntity);
    };
    //审核
    this.auditSeller = function (sellerId, status) {
        return $http.post('../seller/auditSeller/' + sellerId + '/' + status);
    }
});
